src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_bfyx_os_iyx_osv16.cpp

```diff
@@ -197,6 +197,7 @@ ConvolutionKernelBase::DispatchData ConvolutionKernel_bfyx_os_iyx_osv16::SetDefa
 }
 
 KernelsPriority ConvolutionKernel_bfyx_os_iyx_osv16::GetKernelsPriority(const Params& params) const {
+    return DONT_USE_IF_HAVE_SOMETHING_ELSE;
```

Comment:
I hope that's an experimental thing and we're not going to merge that?

Reply:
@vladimir-paramuzov agree. this is just experiment purpose, We need the root cause instead of WA.

Reply:
oh yes. this is to check the status.

Reply:
I just updated the PR to properly resolve the issue.